style.css

```diff
@@ -1,12 +1,10 @@
 body {
-  height: 590px;
+  height: fit-content;
   width: 375px;
   left: 0;
   top: 92px;
```

Comment:
## [OPTIONAL]
💡 This project is mobile-first. You should be resizing your window to see what it would look like on mobile. Currently, you have a set width for the body. This makes your project show a white container on some mobile devices, as shown in the screenshot I made on developer tools for how it would look like on an iPhone XR. You will be adding media queries for the desktop version to render properly in the future. Therefore, you should remove the `width`, as it would be 100%.

Use developer tools to make sure your project looks great on all devices! 

![image](https://user-images.githubusercontent.com/53036416/164837705-b548f4eb-508b-41e3-ab97-1698852c87a7.png)